content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -242,6 +242,12 @@ message ContainerDevices {
 GetAllocatableResources provides information on resources initially available on the worker node.
 It provides more information than kubelet exports to APIServer.
 
+NOTE:
+
+- `GetAllocatableResources` should only be used to evaluate [allocatable](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable) resources on a node. If the goal is to evaluate free/unallocated resources it should 
+be used in conjunction with the List() endpoint. The result obtained by `GetAllocatableResources` would remain the same unless the underlying resources exposed to kubelet change. This happens rarely but when it does (e.g. CPUs onlined/offlined, devices added/removed), client is expected to call `GetAlloctableResources` endpoint.
+
+
```

Comment:
Here we may be able to use the note sidebar (as seen on the [kubernetes test page](https://kubernetes.io/docs/test/#sidebars-and-admonitions)):

```suggestion
{{< note >}}
`GetAllocatableResources` should only be used to evaluate [allocatable](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)
resources on a node. If the goal is to evaluate free/unallocated resources it should be used in
conjunction with the List() endpoint. The result obtained by `GetAllocatableResources` would remain
the same unless the underlying resources exposed to kubelet change. This happens rarely but when
it does (e.g. CPUs onlined/offlined, devices added/removed), client is expected to call
`GetAlloctableResources` endpoint.
{{< /note >}}
```

I've also suggested changing the line breaks as I wasn't sure of the rationale for the existing break.

Reply:
Thanks @nate-double-u! I have fixed the PR based on your suggestion.